ui/css/design-system/colors.scss

```diff
@@ -46,6 +47,7 @@ $color-map: (
   'rinkeby': --rinkeby,
   'goerli': --goerli,
   'localhost': --localhost,
-  'transparent': transparent,
+  'transparent': --transparent,
   'flask-purple': --flask-purple,
+  'inherit': --inherit,
 );
```

Comment:
This should be taken care of in https://github.com/MetaMask/metamask-extension/pull/15507

Reply:
cc @NidhiKJha